### PR TITLE
Consular crashes if a Marathon task has no ports

### PR DESCRIPTION
**Pull request by JayH5**
Related to #29.
We hardcode it... `task['port'][0]`

### CODE REVIEW

**Comment by JayH5** [comment]
Ready for review.

I'm not tackling multiple ports at the moment because that's significantly more complicated. Will leave up to #29.

File: consular/main.py [hunk 1]
```diff
@@ -318,9 +320,20 @@ def register_task_service(self, app_id, task_id, host, port):
...
+        :param list ports:
+            The port numbers the task can be accessed on on the host machine.
         """
+        if not ports:
+            port = None
+        elif len(ports) == 1:
+            [port] = ports
+        else:
+            # TODO: Support multiple ports (issue #29)
+            port = sorted(ports)[0]
```
Comment (smn) [inline consular/main.py]: `max()` ?

File: consular/main.py [hunk 2]
```diff
@@ -318,9 +320,20 @@ def register_task_service(self, app_id, task_id, host, port):
...
+            log.msg('Warning. %d ports found for app "%s". Consular currently '
+                    'only supports a single port. Only the lowest port (%s) '
+                    'will be used.' % (len(ports), app_id, port,))
```
Comment (smn) [inline consular/main.py:335]: Why not a `log.err`? `</nitpick>`

File: consular/main.py (hunk 1 above)
Comment (smn) [inline consular/main.py]: err... `min()`

**Comment by smn** [comment]
:+1: and nice job on splitting it up like this.

File: consular/main.py (hunk 1 above)
Comment (JayH5) [inline consular/main.py]: Oh yeah lol

File: consular/main.py (hunk 2 above)
Comment (JayH5) [inline consular/main.py:335]: To be honest, I'm not really sure where the line between `msg` and `err` lies. If there were a `warn` the decision would be easy.

**Pull request merged**